src/main/java/uk/gov/hmcts/darts/arm/component/impl/ArmResponseFilesProcessSingleElementImpl.java

```diff
@@ -222,6 +229,71 @@ private void processUploadFile(String armInputUploadFilename, List<String> respo
         }
     }
 
+    private void getAndLogCreateRecordFile(ExternalObjectDirectoryEntity externalObjectDirectory, String createRecordFilename) {
+        if (nonNull(createRecordFilename)) {
+            Path jsonPath = null;
+            try {
+                CreateRecordFilenameProcessor createRecordFilenameProcessor = new CreateRecordFilenameProcessor(createRecordFilename);
+                BinaryData createRecordFileBinary = armDataManagementApi.getBlobData(createRecordFilenameProcessor.getCreateRecordFilenameAndPath());
+                if (nonNull(createRecordFileBinary)) {
+                    boolean appendUuidToWorkspace = true;
+                    jsonPath = fileOperationService.saveBinaryDataToSpecifiedWorkspace(
+                        createRecordFileBinary,
+                        createRecordFilenameProcessor.getCreateRecordFilename(),
+                        armDataManagementConfiguration.getTempBlobWorkspace(),
+                        appendUuidToWorkspace
+                    );
+
+                    if (nonNull(jsonPath) && jsonPath.toFile().exists()) {
+                        logResponseFileContents(jsonPath);
+                    }
+                }
+            } catch (Exception e) {
+                log.error("Unable to read create record file {} for EOD {}", createRecordFilename, externalObjectDirectory.getId(), e);
+            } finally {
+                cleanupTemporaryJsonFile(jsonPath);
+            }
+        }
+    }
+
+    private void getAndLogUploadFile(ExternalObjectDirectoryEntity externalObjectDirectory, String uploadFileFilename) {
+        if (nonNull(uploadFileFilename)) {
+            Path jsonPath = null;
+            try {
+                UploadFileFilenameProcessor uploadFileFilenameProcessor = new UploadFileFilenameProcessor(uploadFileFilename);
+                BinaryData uploadFileBinary = armDataManagementApi.getBlobData(uploadFileFilenameProcessor.getUploadFileFilenameAndPath());
+                if (nonNull(uploadFileBinary)) {
+                    boolean appendUuidToWorkspace = true;
+                    jsonPath = fileOperationService.saveBinaryDataToSpecifiedWorkspace(
+                        uploadFileBinary,
+                        uploadFileFilenameProcessor.getUploadFileFilename(),
+                        armDataManagementConfiguration.getTempBlobWorkspace(),
+                        appendUuidToWorkspace
+                    );
+
+                    if (nonNull(jsonPath) && jsonPath.toFile().exists()) {
+                        logResponseFileContents(jsonPath);
+                    }
+                }
+            } catch (Exception e) {
```

Comment:
Can we catch a more specific exception?

Reply:
Again as mentioned above not really relevant as all we want to do is log the contents so specific exceptions are necessary.